tests/unit/helpers/class-wc-helper-product.php

```diff
@@ -56,6 +56,37 @@ public static function create_simple_product( $save = true ) {
 		}
 	}
 
+	/**
+	 * Create simple product.
+	 *
+	 * @since 2.3
```

Comment:
I was sure it work before 🤔 the trial subscriptions problem was fixed on https://github.com/Automattic/woocommerce-payments/pull/4810/commits/672fc7dfedc27ca20f7b126e5237852132c23af7 , also the `2.3` was copied from the function above, I've removed as we are not using the `@since` anymore.